modules/preview-react/form-field/stories/examples/GroupedInputs.tsx

```diff
@@ -0,0 +1,215 @@
+import React from 'react';
+import {FormFieldGroup} from '@workday/canvas-kit-preview-react/form-field';
+import {PrimaryButton, SecondaryButton} from '@workday/canvas-kit-react/button';
+import {system} from '@workday/canvas-tokens-web';
+import {Banner} from '@workday/canvas-kit-react/banner';
+
+import {Checkbox} from '@workday/canvas-kit-react/checkbox';
+import {RadioGroup} from '../../../radio';
+import {createStyles} from '@workday/canvas-kit-styling';
+import {AriaLiveRegion} from '@workday/canvas-kit-react/common';
+
+const formStyles = createStyles({
+  margin: `${system.space.zero} ${system.space.x3}`,
+});
+
+const formButtonStyles = createStyles({
+  display: 'inline-flex',
+  gap: system.space.x2,
+});
+
+const toppings = [
+  {
+    id: 1,
+    label: 'Pepperoni',
+    checked: false,
+  },
+  {
+    id: 2,
+    label: 'Cheese',
+    checked: false,
+  },
+  {
+    id: 3,
+    label: 'Pineapple',
+    checked: false,
+  },
+  {
+    id: 4,
+    label: 'Mushrooms',
+    checked: false,
+  },
+];
+
+const bannerStyles = createStyles({
+  position: 'absolute',
+  right: 0,
+});
+
+export const GroupedInputs = () => {
+  const [toppingsState, setToppingsState] = React.useState(toppings);
+  const [error, setError] = React.useState(undefined);
+  const [radioError, setRadioError] = React.useState(undefined);
+  const [showSuccess, setShowSuccess] = React.useState(false);
+
+  const [value, setValue] = React.useState<string>('');
+  const [formData, setFormData] = React.useState({
+    toppings: [],
+    crust: '',
+  });
+  const handleCheckboxCheck = id => {
+    if (error) {
+      setError(undefined);
+    }
+    setToppingsState(
+      toppingsState.map(item => (item.id === id ? {...item, checked: !item.checked} : item))
+    );
+  };
+
+  const handleRadioChange = (e: React.ChangeEvent) => {
+    if (radioError) {
+      setRadioError(undefined);
+    }
+    const target = e.currentTarget;
+    if (target instanceof HTMLInputElement) {
+      setValue(target.value);
+    }
+  };
+
+  const handleSubmit = e => {
+    e.preventDefault();
+    if (value === '' && toppingsState.every(item => !item.checked)) {
+      setRadioError('error');
+      setError('error');
+    } else if (toppingsState.some(item => !item.checked) && value === '') {
+      setError(undefined);
+      setRadioError('error');
+    } else if (value !== '' && toppingsState.every(item => !item.checked)) {
+      setError('error');
+      setRadioError(undefined);
+    } else {
+      setError(undefined);
+      setRadioError(undefined);
+    }
+    if (
+      error === undefined &&
+      radioError === undefined &&
+      toppingsState.some(item => item.checked) &&
+      value !== ''
```

Comment:
```suggestion
      !error &&
      !radioError &&
      toppingsState.some(item => item.checked) &&
      value
```

you have only 2 options for error: error or undefined
empty string is falsy value